aksparse/client/aksparse/sparse.py

```diff
@@ -199,46 +215,26 @@ def dot(self, input):
             temp_csr = self.to_csr()
             return temp_csr.vector_matrix_dot(input)
         if issubclass(type(input), ak.pdarray):
-            temp_csr = self.to_csr()
-            return temp_csr.matrix_matrix_dot(input)
+            raise NotImplementedError
+            # temp_csr = self.to_csr()
+            # return temp_csr.matrix_matrix_dot(input)
```

Comment:
Is there a reason the old code is commented out here? If not, we should go ahead and delete this.

Reply:
Deleted!